ethereum/executionclient/src/main/java/tech/pegasys/teku/ethereum/executionclient/methods/EthGetBlockByHash.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright ConsenSys Software Inc., 2023
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.ethereum.executionclient.methods;
+
+import java.util.Optional;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.tuweni.bytes.Bytes32;
+import tech.pegasys.teku.ethereum.executionclient.ExecutionEngineClient;
+import tech.pegasys.teku.infrastructure.async.SafeFuture;
+import tech.pegasys.teku.spec.datastructures.execution.PowBlock;
+
+public class EthGetBlockByHash extends AbstractEngineJsonRpcMethod<Optional<PowBlock>> {
+
+  private static final Logger LOG = LogManager.getLogger();
+
+  public EthGetBlockByHash(final ExecutionEngineClient executionEngineClient) {
+    super(executionEngineClient);
+  }
+
+  @Override
+  public String getName() {
+    return "eth_getBlockByHash";
+  }
+
+  @Override
+  public int getVersion() {
+    return Integer.MAX_VALUE;
+  }
```

Comment:
lets just return 1... or better still, 0....

if version 0 then we could just have the versionedName not include and woulldnt have to override...